# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../dfb-linux-fusion/kernel-version.patch
# Copyright (C) 2006 - 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- linux-fusion.git/Makefile.orig	2008-03-26 11:47:55.000000000 +0100
+++ linux-fusion.git/Makefile	2008-03-26 11:49:49.000000000 +0100
@@ -42,9 +42,6 @@
 endif
 
 install: all
-	install -d $(DESTDIR)/usr/include/linux
-	install -m 644 linux/include/linux/fusion.h $(DESTDIR)/usr/include/linux
-
 	install -d $(DESTDIR)$(KERNEL_MODLIB)/drivers/char/fusion
 
 ifeq ($(K_PATCHLEVEL),4)
@@ -54,11 +51,6 @@
 	install -m 644 $(SUB)/fusion.ko $(DESTDIR)$(KERNEL_MODLIB)/drivers/char/fusion
 	rm -f $(DESTDIR)$(KERNEL_MODLIB)/fusion.ko
 endif
-ifneq ($(strip $(DESTDIR)),)
-	/sbin/depmod -ae -b $(DESTDIR) $(KERNEL_VERSION)
-else
-	/sbin/depmod -ae $(KERNEL_VERSION)
-endif
 
 
 
